plugins/sudo/sudo.plugin.zsh

```diff
@@ -12,10 +12,18 @@
 # * Subhaditya Nath <github.com/subnut>
 # * Marc Cornellà <github.com/mcornella>
 # * Carlo Sala <carlosalag@protonmail.com>
+# * Marcelina Hołub <mjholub.me>
 #
 # ------------------------------------------------------------------------------
 
 __sudo-replace-buffer() {
+  #Store the command to use for escalating privileges
+  ROOT_COMMAND=sudo
```

Comment:
It's better to use `zstyle` options for configurability. You can check, for example, `nvm` plugin to see how are we doing this lately 👌🏻